source/templates.tex

```diff
@@ -1390,16 +1388,14 @@
   T& elem(int i) { return v[i]; }
 };
 \end{codeblock}
-
-\pnum
 The prefix
 \tcode{template}
 \tcode{<class}
 \tcode{T>}
 specifies that a template is being declared and that a
 \grammarterm{type-name}
 \tcode{T}
-will be used in the declaration.
+will be used in the definition.
```

Comment:
Yes, this example is a definition; it's a class template definition, too, and it's also a declaration. I'm not sure why you consider "definition" to be the right level of generality. In any event, "declaration" matches "a template is being *declared*"; just saying "definition" here without changing that seems questionable to me.